machine-setup/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:16.04
+
+ADD *.py ./
```

Comment:
Also put this line to the next to the last, otherwise everytime the script update will re-generated the whole image. By putting it in the end the change of script will only rebuild the script related part.